docs/hosts.md

```diff
@@ -175,6 +181,9 @@ server = "https://registry.example.com"
   capabilities = ["pull", "resolve"]
 ```
 
+Omitting the `server =` line will automatically fallback to the intended
```

Comment:
I'm not sure the above example makes a lot of sense to me or it might just be unnecessary. The `server =` is being added to prevent fallback from the mirror to the default host name, however in this case it is unnecessary to even use `host`. I think the current example would try the mirror twice. It is worth mentioning though, as you mention here, that setting `server` in `_default` will prevent the host from getting filled in by the image host.

Maybe it would be clearer to just have two separate examples. One that just adds a global mirror with fallback and another that forces all requests to go through the proxy.

Reply:
Hmm, yes I think you're right! The existing example effectively duplicates it. I'll try to reword it with two examples.